### DISCUSSION TITLE
[Q/A][Java SDK (Component)] how to add customize header when use triple protocol?

### DISCUSSION THREAD

**Discussion by Accelerator96** [apache/dubbo#13943] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

i want add provider svc name to consumer request header.
what else can i do？

version: dubbo 3.2.10
reg: zookeeper

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Just add into Dubbo attachment or use `RpcContext.getClientAttachment()`

**Reply by Accelerator96** [reply]
can set consumer attachment use annotation in dubbo? 
is there any other way ?

**Reply by AlbumenJ** [reply]
Currently we not support that. You can implement a `Filter` by yourself and add the customized attachments.
